### PR TITLE
Added language Powershell

### PR DESCRIPTION
**Pull request by I71**
Powershell errors are pretty specific, but this should get the job done.

### CODE REVIEW

File: Powershell/readme.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# TcSo Powershell
+This is the powershell version of the Try catch stack overflow. Use the try catch block used in [tcso.ps1](tcso.ps1). You can use it in your projects. Please don't forget to make  improvements and submit a new pull request.
+
+#### Requirements
+* Powershell
+* A web browser
+
+#### Credits
+Developed by: [Gautam krishna R](https://github.com/gautamkrishnar/)
```
Comment (gautamkrishnar) [inline Powershell/readme.md]: please update this to your name.

File: Powershell/readme.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+###### Contributors
+* [Asher Dratel](https://github.com/I71)
```
Comment (gautamkrishnar) [inline Powershell/readme.md]: please leave this section empty

**Comment by gautamkrishnar** [comment]
Thanks a lot for opening a PR. I will merge it as soon as you update the credits.

**Comment by I71** [comment]
Should be fixed now, whoops.

**Comment by gautamkrishnar** [comment]
He he he thanks for your contribution @I71 😄

**Pull request merged**